calabash-cucumber/lib/calabash-cucumber/launch/simulator_launcher.rb

```diff
@@ -459,8 +459,6 @@ def ensure_connectivity(app_bundle_path, sdk, device_family, args = nil)
                   begin
                     connected = (ping_app == '200')
                     break if connected
-                  rescue Exception => e
```

Comment:
Good question.  Since the ensure block itself did nothing, I assumed it was safe to remove.